src/extensions/bramble-extensions.json

```diff
@@ -51,6 +51,17 @@
             "extensions/default/InlineColorEditor/img/*.png"
         ]
     },
+    {
+        "path": "extensions/default/InlineBorderRadiusEditor",
+        "less": {
+            "dist/extensions/default/InlineBorderRadiusEditor/css/main.css": [
+                "src/extensions/default/InlineBorderRadiusEditor/css/main.less"
+            ]
+        },
+        "copy": [
+            "extensions/default/InlineBorderRadiusEditor/img/*.png"
```

Comment:
You'll need to also copy your HTML template file.

Reply:
Also your `BorderRadiusProperties.json` file.  Please look at any non-JS files you use, and make sure they get copied.